spec/runners/auth.rb

```diff
@@ -91,7 +91,10 @@ def expected_credential
         end
 
         if credential['mechanism_properties']
-          expected_credential['auth_mech_properties'] = expected_auth_mech_properties
```

Comment:
Can `expected_auth_mech_properties` be removed now?

Reply:
I think so, removed it.